java/core/src/main/java/com/redhat/rhn/manager/rhnpackage/PackageManager.java

```diff
@@ -89,7 +89,7 @@ public class PackageManager extends BaseManager {
     private static final Logger LOG = LogManager.getLogger(PackageManager.class);
 
     // Valid dependency types
-    public static final String[]
+    private static final String[]
```

Comment:
Use `List.of()` and reimplement the loop in `PackageHandler` to use the `List<String>` instead of `String[]`